cmd/kube-apiserver/app/server.go

```diff
@@ -455,12 +456,17 @@ func BuildGenericConfig(s *options.ServerRunOptions, proxyTransport *http.Transp
 		client,
 		sharedInformers,
 		serviceResolver,
-		proxyTransport,
 	)
 	if err != nil {
 		return nil, nil, nil, nil, nil, fmt.Errorf("failed to create admission plugin initializer: %v", err)
 	}
 
+	webhookClientConfig := rest.AnonymousClientConfig(genericConfig.LoopbackClientConfig)
+	if proxyTransport != nil && proxyTransport.Dial != nil {
+		webhookClientConfig.Dial = proxyTransport.Dial
+		// if you set this, you lost e2e safety on your connnection.
+		webhookClientConfig.TLSClientConfig.Insecure = proxyTransport.TLSClientConfig.InsecureSkipVerify
```

Comment:
`proxyTransport.TLSClientConfig` can be nil

Reply:
> proxyTransport.TLSClientConfig can be nil

Not in this path, but I'll gate.